### PR TITLE
Support for Visual Studio for Mac

### PR DESCRIPTION
**Pull request by stephen2m**
Update config to backup installed extensions (update for #1010 )

### CODE REVIEW

**Comment by ivmirx** [comment]
Are the name and #1010 reference correct? Because I see VS Code in the configs and not VS for Mac which is a different product.

**Comment by stephen2m** [comment]
@ivmirx you're actually right....it's indeed VS Code!

**Comment by ivmirx** [comment]
@stephen2m thanks, just double-checking if I need a PR for VS4Mac. :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 ## WIP
 
-- Add support for Micro (via @icopp)
```
Comment (lra) [inline CHANGELOG.md]: Why removing this?

I'll add it back in master.

**Comment by chuckd** [comment]
For anyone confused like I was: this was reverted in June, see #1170.

**Pull request merged**